### DISCUSSION TITLE
I think it's difficult to read the README.md, so I put a discussion here.

### DISCUSSION THREAD

**Discussion by starbrookJ** [BartoszCichecki/LenovoLegionToolkit#1307] [Ideas]
well, it's about a bug i think.

when i unplug the power, toolkit's "Actions" does what it should do. But If i turn off my legion laptop after unplugging the power, then plug in the power, and then turn on my laptop, toolkit doesn't do "Actions". However, I think it should do "Actions", right?

OS: windows 11 pro 23h2
laptop: 5800H 3070

**Comment by starbrookJ** [comment]
Also, thanks to all the coders who created this toolkit!

**Comment by BartoszCichecki** [comment]
AC related actions should run on startup (as the only ones unless combine with "On startup" action)
